Slack.Automation/Promact.Core.Repository/ScrumRepository/ScrumBotRepository.cs

```diff
@@ -773,53 +779,55 @@ private bool IsPrivateChannel(string slackChannelId)
         private async Task<string> GetQuestionAsync(int scrumId, List<Question> questions, List<User> users, int projectId)
         {
             List<ScrumAnswer> scrumAnswers = _scrumAnswerDataRepository.FetchAsync(x => x.ScrumId == scrumId).Result.ToList();
+            User user = new User();
```

Comment:
If I do not assign then I get the following error, "User of unassigned local variable _'user'_"